n-heptane/Burcat 1981/st_burcat_1981-1.yaml

```diff
@@ -1,7 +1,8 @@
-file-author:
+file-authors:
     name: Kyle Niemeyer
+    name: Bradley Carrano
```

Comment:
I didn't notice this yesterday, but there should be a dash in front of each of these,
```
  - name: Kyle Niemeyer
  - name: Bradley Carrano
```